src/goblintCil.ml

```diff
@@ -0,0 +1,45 @@
+include Cil
+(** @inline *)
+
+(** {1 CIL modules} *)
+
+module Cfg = Cfg
+module Check = Check
+module Cil = Cil
+module Cilint = Cilint
+module Cillower = Cillower (* ? *)
+module Ciloptions = Ciloptions (* ? *)
```

Comment:
There are some question marks here, because now with this wrapper module, we have a choice which modules we want to expose (as submodules of `GoblintCil`) and which are completely internal. Most are needed from somewhere outside (extensions or Goblint), but a handful seem to be very internal things, so we could hide them.